src/explorer/ApisTreeItem.ts

```diff
@@ -67,43 +68,72 @@ export class ApisTreeItem extends AzureParentTreeItem<IServiceTreeRoot> {
             });
     }
 
-    public async createChildImpl(showCreatingTreeItem: (label: string) => void, userOptions?: { apiName: string, document?: IOpenApiImportObject }): Promise<ApiTreeItem> {
-        if (userOptions && userOptions.document && userOptions.apiName) {
-            const apiName = userOptions.apiName;
+    public async createChildImpl(showCreatingTreeItem: (label: string) => void, userOptions?: { apiName: string, document?: IOpenApiImportObject, apiContract?: ApiContract }): Promise<ApiTreeItem> {
+        if (userOptions) {
+            if (userOptions.document) {
+                return await this.createApiWithDocument(showCreatingTreeItem, userOptions.apiName, userOptions.document);
```

Comment:
Please test the exisiting flows work as expected

Reply:
ok, tested it